showcase/image_generation/dcgan_mnist.py

```diff
@@ -92,10 +92,10 @@ def forward(self, data, state):
         return data
 
 
-def get_estimator(batch_size=256, epochs=10):
+def get_estimator(batch_size=256, epochs=50):
     # prepare data
-    (x_train, _), (x_eval, _) = tf.keras.datasets.mnist.load_data()
-    data = {"train": {"x": np.expand_dims(x_train, -1)}, "eval": {"x": np.expand_dims(x_eval, -1)}}
+    (x_train, _), (_, _) = tf.keras.datasets.mnist.load_data()
```

Comment:
if we're not doing eval do we want to add that data into the train set?

Reply:
I think we will do the same as tensorflow tutorial.   no need to add that.